torch/quantization/fx/quantization_patterns.py

```diff
@@ -841,7 +841,12 @@ class FixedQParamsOpQuantizeHandler(QuantizeHandler):
     def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 is_reference: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
-        return quantizer.quantized_graph.node_copy(node, load_arg(quantized=None))
+        qconfig = quantizer.qconfig_map[node.name]
+        dtypes = get_qconfig_dtypes(qconfig)
+        if dtypes == (torch.float16, torch.float16, None):
+            return quantizer.quantized_graph.node_copy(node, load_arg(quantized=False))
+        else:
+            return quantizer.quantized_graph.node_copy(node, load_arg(quantized=None))
```

Comment:
just for my own understanding, what's the difference between `load_arg(quantized=False)` and `load_arg(quantized=None)`?

Reply:
load_arg(quantized=False) means dequantize the node if needed, load_arg(quantized=None) means do nothing, just copy the node